### DISCUSSION TITLE
PACE-tomo workflow

### DISCUSSION THREAD

**Discussion by asarnow** [nextpyp/.github#4] [Ideas]
First, let me say thank you, PYP is awesome and IMO is game changing for the cryo-ET software landscape. 

I was wondering if there is an existing workflow for parallel tilt series data from PACE-tomo?

My files from PACE-tomo don't quite match the TILTSERIES_SCANORD_ANGLE pattern, because each parallel tilt series is interleaved in SCANORD, with a shared TILTSERIES (or really, "TARGET") label. For example with 41 tilts and 5 parallel tilt series, the SCANORD goes from 1 to 205, with e.g. 1, 42, 83, etc. identifying the first image in each series. They are always completely uniform, so splitting them modulo the number of tilts is reliable/easy. There are also .mdoc files for each TS, but I am not sure how to use these with PYP. There's no place to specify a directory for .mdoc files and placing them alongside the movies doesn't seem to work.

**Comment by abartesaghi** [comment]
Thanks @asarnow, you are right, the current version of nextPYP will not read these PACE-tomo .mdoc files because the tilt-series names do not match the name of the frames. This is a simple change though, so we will include this functionality in the next maintenance version of nextPYP which is scheduled to be released in the next couple of days.

**Reply by abartesaghi** [reply]
This feature has been implemented in [v0.5.1](https://nextpyp.app/files/pyp/0.5.1/docs/index.html). All frames and mdoc files should be placed in the same folder and the option `Read metadata from mdoc files` should be checked (Frame alignment tab, Pre-processing block).
